sokutei.h

```diff
@@ -0,0 +1,188 @@
+#ifndef SOKUTEI_BENCHMARK_H
+#define SOKUTEI_BENCHMARK_H
+
+#define SOKUTEI_ITERATIONS 10
```

Comment:
I think we should decide on an API regarding the iterations. 90% of our users should be able to get a benchmark up and running without defining a single macro. Thus, in my opinion, for the number of measured iterations, we should prefer a runtime setting through some method call.

As a consequence, the name of this macro should be more like `SOKUTEI_MAX_MEASURED_ITERATIONS`.

What do you think?

Of course, if the user sets the number of measured iterations to be more than the maximum value, there will be problems that should be resolved somehow.